aws/random_test.go

```diff
@@ -0,0 +1,55 @@
+package aws
```

Comment:
these methods look great, my only confusion was the filename since its `_test` ..do you think we could use a different filename or is it best for long-term usage?

Reply:
Putting the `_test` on the end puts anything in the file in a pseudo package that's only built at test time. We may eventually want to put it in a testing package or try to get it added to the Plugin SDK